broker/user_provided/controller/controller.go

```diff
@@ -26,26 +26,27 @@ import (
 	sbmodel "github.com/kubernetes-incubator/service-catalog/model/service_broker"
 )
 
-type UserProvidedServiceInstance struct {
+type userProvidedServiceInstance struct {
 	Name       string
 	Credential *sbmodel.Credential
 }
 
-type Controller struct {
-	instanceMap map[string]*UserProvidedServiceInstance
+type userProvidedController struct {
+	instanceMap map[string]*userProvidedServiceInstance
 }
 
 // Verify that Controller implements the broker Controller interface.
-var _ controller.Controller = (*Controller)(nil)
+var _ controller.Controller = (*userProvidedController)(nil)
```

Comment:
verifying that the `*userProvidedController` conforms to `Controller` should be left to test code. also, note that the code in `CreateController` does so anyway